src/components/MintingInfo/MintingInfo.component.tsx

```diff
@@ -0,0 +1,38 @@
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import React from 'react';
+import { MintingInfoPros } from './MintingInfo.types';
+
+export const MintingInfo = ({ data }: MintingInfoPros) => (
+  <>
+    {data.map((d, i) => (
+      <Box
+        key={i}
+        sx={{
+          display: 'grid',
+          gridTemplateColumns: 'repeat(2, 1fr)',
+          gap: 2,
+          my: '5px',
+        }}
+      >
+        <Typography
+          variant="h4"
+          sx={{
+            textAlign: 'right',
+          }}
+        >
+          {d.label} :
```

Comment:
I think we should't have this space here
```suggestion
          {d.label}:
```